.travis.yml

```diff
@@ -14,4 +14,18 @@ before_script:
   - julia -e 'using Pkg; Pkg.add(PackageSpec(name="Compose", rev="master"))'
```

Comment:
Hmm.. the dependency in `docs/Project.toml` overrides this. Is it important that Compose is on `master` for documentation building as well?

Reply:
in general, yes.  sometimes changes to gadfly master depend on changes to compose master that haven't been tagged yet.

Reply:
Alright. I did not see how I could do this without duplication, so I just added the same `Pkg.add` into the docs building command.